Gemfile

```diff
@@ -1,6 +1,8 @@
 source "https://rubygems.org"
 
 gem "fastlane"
+gem "fastlane-plugin-diawi"
```

Comment:
I think those aren't necessary because later in the file it reads from `Pluginfile` where they exist.

Reply:
I see, thanks